kitsune/products/jinja2/products/product.html

```diff
@@ -81,7 +81,7 @@ <h1 class="product-title cf">
   <div class="grid_12">
     <section id="help-topics">
       <h1>{{ _('Help Topics') }}</h1>
-      {{ help_topics(topics, show_community_support=product.questions_enabled(request.LANGUAGE_CODE)) }}
+      {{ help_topics(topics, show_community_support) }}
```

Comment:
This code works, at least when running it locally, I see it working properly. The show_community_support "variable", or how is it called in templates, is being passed to `product.html` from `kitsune/products/views.py`, so it shouldn't be needed to `show_community_support=show_community_support` the same as it's not needed to `topics=topics`.